src/renderer/widgets/AddProxyModal/ui/Confirmation.tsx

```diff
@@ -170,6 +171,7 @@ export const Confirmation = ({ onGoBack, secondaryActionButton }: Props) => {
         <div className="flex gap-4">
           {secondaryActionButton}
           <SignButton
+            isDefault={!isEmptyComponent(secondaryActionButton)}
```

Comment:
Why do we try to check inner content of a `secondaryActionButton`?

Reply:
Because for add to basket button we check availability inside of component and If component return null, it still be a component